experiments/ClimaEarth/user_io/debug_plots.jl

```diff
@@ -75,7 +75,7 @@ function plot_global_conservation(
     if !softfail
         @info typeof(cc)
         @info rse[end]
-        @assert rse[end] < 0.0065
+        @assert rse[end] < 0.035
```

Comment:
Do we need to make this threshold visible in the docs ? e.g. leaderboard / diagnostic output section?

Reply:
To me, this value is more for internal use as we develop the coupled model. E.g. I expect it will change as we change our regridding approach moving forward, so I'm not sure we need to broadcast it yet. But if you think it would be useful, I can add it to the docs too

Reply:
Yeah since this is just in `debug_plots` we perhaps don't need to include this in docs.